styles.css

```diff
@@ -79,13 +83,27 @@ label {
   visibility: hidden;
 }
 
+#clear-btn {
+  font-family: 'Quicksand', sans-serif;
+  font-size: 16px;
+  background: #134d71;
+  color: #ffffff;
+  display: block;
+  border: 0;
+  border-radius: 6px;
+  width: 50px;
+  height: 28px;
+  align-self: baseline;
+}
+
```

Comment:
Assuming you want this button in the upper right hand corner of the message container, you may want to try setting the container to `position: relative` and the button to `position: absolute` and then play around with the `top` and `right` properties plus `margin` to get it where you want it. Check out [the docs](https://developer.mozilla.org/en-US/docs/Web/CSS/position) on the `position` property to see how `absolute` and `relative` can work together to achieve what you want here.